src/components/fields/image-upload/image-review/image-thumbnails/image-thumbnails.styles.ts

```diff
@@ -92,14 +95,24 @@ export const HiddenFileSelect = styled.input`
 	display: none;
 `;
 
-export const AddImageButton = styled(IconButton)`
+export const AddImageButton = styled(IconButton)<{ theme: ThemeSpec }>`
```

Comment:
not necessary to declare theme prop; styled-components inherit the theme prop by default

Reply:
removed